packages/medusa/src/api/routes/admin/batch/confirm-batch-job.ts

```diff
@@ -0,0 +1,31 @@
+import { BatchJobService } from "../../../../services"
+
+/**
+ * @oas [post] /batch/{id}/confirm
+ * operationId: "PostBatchBatchComfirm"
```

Comment:
comment: See comment on consistency in operation ids
```suggestion
 * operationId: "PostBatchJobsBatchJobConfirm"
```